plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/metadata/MetadataRebuilder.java

```diff
@@ -238,23 +233,11 @@ public void end() {
      * Process exits from group level, executed in isolation.
      */
     private void rebuildMetadataExitGroup(final String currentGroupId) {
-      storageFacet.perform(db, new Operation<Void>()
```

Comment:
Curious, what is the retry scope now after these changes of MetadataRebuilder and this method? Will it retry the _whole operation_?